WcaOnRails/app/models/user.rb

```diff
@@ -1284,6 +1298,7 @@ def delegate_role
       is_active: true,
       group: self.region,
       user: self,
+      is_lead: delegate_status == "senior_delegate",
```

Comment:
Do we have enumerated constants for Delegate status available somewhere?

Reply:
Yes we do, added now. Sorry for missing it. :-)